src/components/layouts/collection-page-layout.tsx

```diff
@@ -232,14 +233,22 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
     ? sanityIllustrator
     : illustrator
 
+  const courseDependenciesWithData = !isEmpty(sanityDependencies)
+    ? sanityDependencies
+    : dependencies
+
   const podcast = first(
     course?.items?.filter((item: any) => item.type === 'podcast'),
   )
 
   logCollectionResource(course)
 
   const courseTags = tags.map((tag: any) => {
-    const version = get(dependencies, tag.name)
+    const ogVersion = get(dependencies, tag.name)
+    const sanityTag = find(sanityDependencies, {name: tag.name}).version
+
+    const version = !isEmpty(sanityTag) ? sanityTag : ogVersion
```

Comment:
The sanity data is shaped a little different than how we were hard coding it in `courseDependencies.json` so I'm using find to grab the right dependency that matches the tag name then pulling the version off of that dependency.

`version` now switches on if a `sanityTag` is present or not